bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -1539,6 +1539,8 @@ PrefPage_Import_Checkbox_CreateTourIdWithTime_Tooltip = The unique tour ID can b
 PrefPage_Import_Checkbox_IgnoreInvalidFiles           = Ignore &invalid files when using the easy import
 PrefPage_Import_Checkbox_IgnoreInvalidFiles_Tooltip   = When using the Easy Import, invalid files will be saved (their absolute path and file names will be saved in the file \
                                                         "invalidfiles_to_ignore.txt") so that they are not considered files to be imported in the future.
+PrefPage_Import_Checkbox_SetBodyWeight                = Set the person's weight when saving an imported a tour

```

Comment:
"Set the person's weight when saving an imported a tour"

Is the "a" before "tour" correct? I would remove the "a"

When I translate the above text into german and then back into english (with google) then the text below is returned

"Set the weight of the person when saving an imported tour"

Reply:
Thanks for the catch. Fixed in the upcoming PR